db/pre-re/status.yml

```diff
@@ -214,6 +219,7 @@ Body:
     Fail:
       Refresh: true
       Inspiration: true
+    EndReturn:
```

Comment:
I just tested Confusion and it should not be removed on damage. Only causing Confusion again removes it.